api/app/hfi/hfi_calc.py

```diff
@@ -328,15 +390,18 @@ def calculate_mean_intensity(dailies: List[StationDaily]):
     return math.ceil(mean_intensity_group)
```

Comment:
my spidey senses have been alerted by this, although I could be wrong - we used to have logic on the frontend for rounding MIG values down to x when MIG was <= x.7, and rounding up to x+1 otherwise. I think we've now removed that frontend logic to do all the calculations on the backend, so `math.ceil()` might not be good enough here??

Reply:
I think that's exactly what's being done here? `if >= x.8` then `math.ceil(x)` - which `==` math.flor(x)+1`

PS. Not sure why this is being caught now? I didn't touch this code. This is code you wrote.

Reply:
well if I wrote it, then it must be perfect! /s 
my concern is that `math.ceil(x.7)` will be rounded up to `x+1`, when it should be left at `x`. I'll double-check the logic in your dev env later